### PR TITLE
Fix Python 3 errors

### PR DESCRIPTION
**Pull request by saschanaz**
As Python 2 will die in 2020.

### CODE REVIEW

**Comment by kainino0x** [comment]
If this is python3-only, the `#!` needs to point at python3 explicitly, as Jeff pointed out.

Even if not, I don't think we want to maintain a polyglot, so it should point at one or the other explicitly.

**Comment by saschanaz** [comment]
@kainino0x The CI only have Python 2.7, and I'm not sure whether Travis can install both Py2/3. Can it?

**Comment by saschanaz** [comment]
Hmm, maybe we could just replace 2 with 3. I'll try it.

File: specs/latest/1.0/extract-idl.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
```
Comment (jdashg) [inline specs/latest/1.0/extract-idl.py]: `#! /usr/bin/env python3` not `#! /usr/bin/python3`.

**Comment by jdashg** [comment]
@kenrussell Please double-check that we're good to upgrade to py3 on Travis. Otherwise LGTM!

**Pull request merged**